src/common/utils.tsx

```diff
@@ -16,6 +18,27 @@ export function readFileAsJson<T>(file: File): Promise<T> {
   });
 }
 
+export function readFileAsCsv<T>(file: File): Promise<T> {
+  return new Promise((resolve, reject) => {
+    const reader: FileReader = new FileReader();
+    if (reader.error) {
+      reject(reader.error);
+    }
+    reader.onload = () => {
+      try {
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const c: any = csv()
```

Comment:
why not passing directly the result instead of using the worst variable name with the worst type ?

Also it could be correctly types right =)

Reply:
HAHAHAHA omg I forgot to change!! thanks for pointing it out!

Reply:
ok I've changed this to use `T[]` instead